views/components/Sidebar/SidebarLink.css

```diff
@@ -17,3 +20,11 @@
   background-color: var(--primary-light);
   border-radius: 8px;
 }
+
+.SidebarLink_link-active .MuiListItemIcon-root {
+  color: #054DA7;
```

Comment:
is this color a variable already..?

Reply:
it was in the JS theme but not css. I added it here https://github.com/AMP-SCZ/dpdash/pull/474/commits/6c83b9f38f8762e6017ff936e2889f28e22016d7